lms/djangoapps/discussion/signals/handlers.py

```diff
@@ -41,7 +41,9 @@ def update_discussions_on_course_publish(sender, course_key, **kwargs):  # pylin
 
 
 @receiver(signals.comment_created)
-def send_discussion_email_notification(sender, user, post, **kwargs):  # lint-amnesty, pylint: disable=missing-function-docstring, unused-argument
+@receiver(signals.comment_flagged)
```

Comment:
might be cleaner if we have a separate signal receiver method for flagged content